### PR TITLE
Add OSNAP Arctic Transects transport time series

### PR DESCRIPTION
**Pull request by anirban89**
Add Arctic Transects from branch https://github.com/milenaveneziani/geometric_features/tree/flipOSNAP_renameArcticTransectsTag of geometric features.
Separate all arctic transects from standard transects using transectGroup flag in config option `timeSeriesTransport`.
When running `mpas_analysis` without the flag `--polar_regions` only standard transects are plotted. All Arctic transects including the three new ones "Hudson Bay-Labrador Sea", "OSNAP section East" and "OSNAP section West"
are only plotted when `--polar_regions` is specified.
timeSeriesTransport split into transportGroups "Transport Transects" and "Arctic Transport Transects". By default all Arctic transects are removed from standard transects. 

User can specify the list of standard transects to plot with transectsToPlot under config option (default.cfg) `timeSeriesTransportTransects` and leave it as `all` under `timeSeriesArcticTransportTransects`. List can not include an Arctic Transect.
Similarly user can also specify the list of arctic transects to plot with transectsToPlot under config option `timeSeriesArcticTransportTransects` but to plot these `mpas_analysis` needs to be run with the `--polar_regions` flag.

Files changed: `default.cfg`, `polar_regions.cfg` and `ocean/time_series_transport.py`
@xylar @milenaveneziani @alicebarthel

### CODE REVIEW

**Comment by xylar** [comment]
@anirban89, I think this still has the default title from the name of the branch.  Could you change it to be a little more informative?

**Comment by xylar** [comment]
This will need https://github.com/MPAS-Dev/geometric_features/pull/185 to be merged, a new release of `geometric_features`, and MPAS-Analysis to be updated to that new version.  This isn't as big a deal as it sounds but I'd like to hold off on that process so we can do a bit more testing of https://github.com/MPAS-Dev/geometric_features/pull/185.

**Comment by anirban89** [comment]
@xylar  yes I agree this PR does not need to be merged right now

**Comment by xylar** [comment]
@alicebarthel and @milenaveneziani, we want to add obs. to this.  What is the right upper and lower value we should use?  Min and max in time?

**Comment by milenaveneziani** [comment]
@xylar: sorry, I never replied to this. I usually compute a standard deviation from all the obs I have (or get it from previous literature) and then plot that as +-sigma around the mean value. @alicebarthel: what do you think?

**Comment by cbegeman** [comment]
@xylar I have a commit that adds the observational range to this PR (and I've rebased onto develop). After that, this PR should be ready to merge. How should I proceed? I am not able to push to @anirban89's fork.

**Comment by cbegeman** [comment]
@xylar @milenaveneziani @alicebarthel This is ready for review. I don't think all of you need to review it, just maybe whichever one of you has time and interest.

**Comment by xylar** [comment]
I'm running my suite of tests on this now.  Thanks for reviving it @cbegeman!

**Comment by xylar** [comment]
It seems like the docs may need a bit of an update:
https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/docs/users_guide/tasks/timeSeriesTransport.rst

**Comment by alicebarthel** [comment]
Thanks @cbegeman! I probably won't get to it today but hopefully tomorrow.

**Review by xylar — APPROVED** [review]
Other than the docs, I think this is ready to go.  Here are some test results at QU240 res:
https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.xasay-davis/analysis_testing/chrysalis/add_osnap_voltrans_ts/main_py3.11/ocean/index.html#transporttime

Here are the full test results, which all seem okay:
https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.xasay-davis/analysis_testing/chrysalis/add_osnap_voltrans_ts/

**Review by alicebarthel — APPROVED** [review]
Thanks @cbegeman, these look great! (the tasks and code, not the sad transports through osnap)
Approving based on visual inspection and a short 10-yr analysis test, provided the minor doc updates that Xylar already pointed at.

**Comment by cbegeman** [comment]
@xylar This is ready for you to merge when you get back from vacation.

**Comment by xylar** [comment]
Thanks @anirban89, @cbegeman, @alicebarthel and @milenaveneziani!

**Pull request merged**
